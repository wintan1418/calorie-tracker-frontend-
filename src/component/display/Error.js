export default function Error() {
  return (
    <div>
      <h1>ohh! seems page is ghosted</h1>
    </div>
  );
}
