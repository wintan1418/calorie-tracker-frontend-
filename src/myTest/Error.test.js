import React from 'react';
import renderer from 'react-test-renderer';
import Error from '../component/display/Error';

it ('renders accurately', () => {
  const tree = renderer.create(<Error />).toJSON();
  expect(tree).toMatchSnapshot();
}

)
