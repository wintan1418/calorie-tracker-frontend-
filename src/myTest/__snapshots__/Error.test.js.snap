// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders accurately 1`] = `
<div>
  <h1
    className="wait"
  >
    ohh! seems page is ghosted
  </h1>
</div>
`;
